package.json

```diff
@@ -20,7 +20,18 @@
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-prettier": "^3.3.1",
     "eslint-plugin-react": "^7.22.0",
+    "husky": "^4.3.8",
+    "lint-staged": "^10.5.4",
     "prettier": "^2.2.1",
     "typescript": "^4.2.3"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx,md,mdx,yml,json}": "prettier --write",
```

Comment:
`yaml` isn't included here. Is this intentional (and trying to set a standard of using `yml`?

Reply:
Nope, I'll add `yaml`. Was just following the pattern that the only yaml files in the project currently are `yml` but they're both totally fine.